test/e2e/specs/editor/editor__page-basic-flow.ts

```diff
@@ -27,9 +28,19 @@ describe( DataHelper.createSuiteTitle( 'Editor: Basic Post Flow' ), function ()
 	let pagesPage: PagesPage;
 	let editorSettingsSidebarComponent: EditorSettingsSidebarComponent;
 	let publishedUrl: URL;
-	const accountName = envVariables.GUTENBERG_EDGE
-		? 'gutenbergSimpleSiteEdgeUser'
-		: 'simpleSitePersonalPlanUser';
+
+	const accountName = getTestAccountByFeature(
+		{
+			gutenberg: envVariables.GUTENBERG_EDGE ? 'edge' : 'stable',
+			siteType: envVariables.TEST_ON_ATOMIC ? 'atomic' : 'simple',
+		},
+		[
+			// TODO This could be moved over to the default accounts in calypso-e2e/lib/utils/get-test-account-by-feature.ts provided we add
+			// a variation: 'siteEditor' to it. For now, it just replaces the default ones that match the same criteria when they're passed here.
+			{ gutenberg: 'stable', siteType: 'simple', accountName: 'gutenbergSimpleSiteEdgeUser' },
```

Comment:
I think I've made an oversight here and passing `{ gutenberg: 'stable', siteType: 'simple', accountName: 'gutenbergSimpleSiteEdgeUser' }` is not needed (since it's already defined as part of the [default criteria](https://github.com/Automattic/wp-calypso/pull/61783/files#diff-9002489e27625a1d0b7fa59a8f40b5fdafe17d31b263fa5511df77ab88f3edacR34-R56)). In fact, it'd be better to just have everything defined in there and I think I'll just do it.